comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/DataDog/datadog-agent/comp/metadata/internal/util"
 	iainterface "github.com/DataDog/datadog-agent/comp/metadata/inventoryagent"
 	"github.com/DataDog/datadog-agent/comp/metadata/runner/runnerimpl"
+	"github.com/DataDog/datadog-agent/pkg/collector/python"
```

Comment:
This adds a hard dependency to the python collector which is not doable in this payload. The Agent payload is used by every flavor of the agent, including those who don't ship python.

A better solution would be to move this to `inventorychecks`, add a method `SetPackage` similar to [this](https://github.com/DataDog/datadog-agent/blob/main/comp/metadata/inventorychecks/component.go#L17) and use it from the python package to populate the payload.

Reply:
I can definitely move it to the inventorychecks